vsintegration/ProjectTemplates/NetCore259Project/Template/PortableLibrary.fsproj

```diff
@@ -41,7 +41,7 @@
       <HintPath>$(MSBuildProgramFiles32)\Reference Assemblies\Microsoft\FSharp\.NETCore\$(TargetFSharpCoreVersion)\FSharp.Core.dll</HintPath>
     </Reference>
     <Reference Include="System.ValueTuple">
-      <HintPath>..\packages\System.ValueTuple.4.3.0\lib\netstandard1.0\System.ValueTuple.dll</HintPath>
```

Comment:
I'm afraid that tuples will need to stay as they are for now.  There is some VSIX packaging of nugets that happens.  A restore basically happens at project creation eliminating a wiggly error.